download.sh

```diff
@@ -1,181 +1,25 @@
-    /assets/images/track/cars/blue.png
-    /assets/images/common/logos/chrome-experiment.png
-    /assets/images/common/logos/chrome.png
-    /assets/images/common/close.png
-    /assets/images/lineup/dotted-line.png
-    /assets/images/common/devices/droid-razr.png
-    /assets/images/common/places/empty.png
-    /assets/images/lineup/fade-left.png
-    /assets/images/lineup/fade-right.png
-    /assets/images/common/devices/galaxy-nexus.png
-    /assets/images/track/loader/glow.png
-    /assets/images/track/touchglow/green.png
-    /assets/images/track/cars/green.png
-    /assets/images/common/devices/ipad-mini.png
-    /assets/images/common/devices/ipad.png
-    /assets/images/common/devices/iphone-4.png
-    /assets/images/common/devices/iphone-5.png
-    /assets/images/track/name-bg.png
-    /assets/images/common/devices/nexus-4.png
-    /assets/images/common/devices/nexus-7.png
-    /assets/images/common/devices/nexus-10.png
-    /assets/images/track/touchglow/orange.png
-    /assets/images/track/cars/orange.png
-    /assets/images/common/logos/play.png
-    /assets/images/common/logos/racer-large-mobile.png
-    /assets/images/common/logos/racer-large.png
-    /assets/images/common/logos/racer-small.png
-    /assets/images/common/logos/racer.png
-    /assets/images/common/spin-loader.png
-    /assets/images/track/touchglow/red.png
-    /assets/images/track/cars/red.png
-    /assets/images/common/devices/samsung-epic-4g.png
-    /assets/images/common/devices/samsung-galaxy-ace.png
-    /assets/images/common/devices/samsung-galaxy-note-2.png
-    /assets/images/common/devices/samsung-galaxy-s2.png
-    /assets/images/common/devices/samsung-galaxy-s3.png
-    /assets/images/common/devices/samsung-galaxy-tab-2.png
-    /assets/images/common/devices/samsung-galaxy-y.png
-    /assets/images/track/loader/scan.png
-    /assets/images/track/banner/shadow.png
-    /assets/images/common/logos/share.png
-    /assets/images/track/startline.png
-    /assets/images/lineup/tick.png
-    /assets/images/track/loader/track.png
-    /assets/images/track/win-outline.gif
-    /assets/images/track/touchglow/yellow.png
-    /assets/images/track/cars/yellow.png
-    /assets/images/lineup/you-triangle-dark.png
-    /assets/images/lineup/you-triangle.png
-    /assets/images/content/hands.jpg
-    /assets/images/common/logos/youtube.png
-    /assets/images/common/logos/racer-fallback.png
-    /assets/images/common/logos/chrome-small.png
-    /assets/images/lineup/you-triangle.png
-    /assets/sprites/crash/blue.png
-    /assets/sprites/crash/green.png
-
+mkdir -p client
+
+# Download website
+(
+    cd client &&
+    # nH removes the hostname, cut-dirs removes the racer subdirectory
+    wget -r --no-parent --compression=auto --no-cookies -nH --cut-dirs=1 https://chrome.com/racer/index.html &&
+    wget -r --no-parent --compression=auto --no-cookies -nH --cut-dirs=1 --base=https://chrome.com/racer/ --input-file=../extra_resources.txt
```

Comment:
Thanks for improving my code, this is much better!

Maybe `../` should be removed from `--input-file=../extra_resources.txt`?

Reply:
That one's intentional, it's in the client folder at that moment :D

Reply:
Oh wow, true 😅  Thanks!